index.rst

```diff
@@ -115,6 +115,22 @@ Use the ``--format`` option if you want to use a custom format string. Both of
 these arguments along with ``--ci`` argument run coala in non-interactive mode
 which is suitable for continuous integration.
 
+Hint
+~~~~
+
+``--json`` makes the coala output easy to read and parse. It consists of
+a collection of name/value pairs usually represented by objects and an ordered
+list of values stored in arrays. You can read more about this format `here
+<http://www.json.org>`__.
+
+
+
+``--format`` returns a linear, one line output. The separator for its elements
+will be ``:``. This option will not show all tested areas but those with
```

Comment:
it won't necessarily be separated by colon, you can give a custom format string, this is the easiest to parse as you can customize the output heavily